src/pages/Home/Home.js

```diff
@@ -1,13 +1,13 @@
 import React from "react";
+import Channels from "../../components/Channels/Channels";
 
-const Home = () => {
+export const Home = () => {
   return (
     <div>
       <header>
```

Comment:
I would removed the `header` tag, it's an unnecessary DOM element. You want to minimize the number of elements if you can. Helps with space and rendering.